tox.ini

```diff
@@ -7,3 +7,4 @@ deps =
     pre-commit
 commands =
     pre-commit run --all-files
+    pytest webhook_server_container/tests/test_github_api.py
```

Comment:
`pytest webhook_server_container/tests/test_github_api.p` should be under new testenv called `unittests`